File: Signal.CPP/IntroDesign.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "IntroDesign.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma resource "*.dfm"
TIntroDesignForm *IntroDesignForm;
//---------------------------------------------------------------------------
__fastcall TIntroDesignForm::TIntroDesignForm(TComponent* Owner)
	: TBasicForm3(Owner)
{
	RichEdit1->Lines->Clear();
	RichEdit1->DefAttributes->Size = 12;
	RichEdit1->DefAttributes->Style = RichEdit1->DefAttributes->Style << TFontStyle::fsBold;
	RichEdit1->SelAttributes->Color = (TColor) RGB(51,51,153);
	RichEdit1->SelAttributes->Size = 14;
	RichEdit1->Lines->Add("Architecture of the library is organized in three layers: ");
	RichEdit1->Lines->Add("");
	RichEdit1->Paragraph->Numbering = nsBullet;
	RichEdit1->Paragraph->FirstIndent = 10;
	RichEdit1->Lines->Add("Complete set of signal processing routines based on MtxVec.");
	RichEdit1->Lines->Add("With a set of components based on signal processing routines "
		" your application gets a jump start in user interface and processing power.");
	RichEdit1->Lines->Add("Connectivity layer gives you the capability to visually programm " 
		"DSP algorithm's by connecting components.");
	RichEdit1->Paragraph->Numbering = nsNone;
	RichEdit1->Paragraph->FirstIndent = 0;

}
//---------------------------------------------------------------------------

File: Signal.Delphi/IntroDesign.pas
unit IntroDesign;

interface

uses
  Windows, Messages, SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
  Basic3, StdCtrls, ComCtrls;

type
  TIntroDesignForm = class(TBasicForm3)
    procedure FormCreate(Sender: TObject);
  private
    { Private declarations }
  public
    { Public declarations }
  end;

var
  IntroDesignForm: TIntroDesignForm;

implementation

{$IFDEF CLR}
{$R *.nfm}
{$ELSE}
{$R *.dfm}
{$ENDIF}

procedure TIntroDesignForm.FormCreate(Sender: TObject);
begin
  inherited;
  With RichEdit1.Lines, RichEdit1 do
  begin
    Clear;
    DefAttributes.Size := 10;
    DefAttributes.Style := [fsBold];

    SelAttributes.Size := 12;
    SelAttributes.Color := RGB(51,51,153);
    Add('Architecture of the library is organized in three ' +
        'layers: ');
    Add('');
    Paragraph.FirstIndent := 10;
    Paragraph.Numbering := nsBullet;
    Add('Complete set of signal processing routines based on MtxVec.');
    Add('With a set of components based on signal processing routines ' +
        ' your application gets a jump start in user interface and processing power.');
    Add('Connectivity layer gives you the capability to visually programm ' +
        'DSP algorithm''s by connecting components.');
    Paragraph.FirstIndent := 0;
    Paragraph.Numbering := nsNone;
  end;
end;

initialization
   RegisterClass(TIntroDesignForm);

end.
